junit-jupiter-params/junit-jupiter-params.gradle

```diff
@@ -47,6 +47,9 @@ dependencies {
 	testRuntimeOnly(project(':junit-platform-console'))
 	testRuntimeOnly("org.apache.logging.log4j:log4j-core:${log4jVersion}")
 	testRuntimeOnly("org.apache.logging.log4j:log4j-jul:${log4jVersion}")
+
+	compileOnly("org.jetbrains.kotlin:kotlin-stdlib:${kotlinVersion}")
+	testCompile("org.jetbrains.kotlin:kotlin-test-junit:${kotlinVersion}")
```

Comment:
Please replace this line with
```
testImplementation("org.jetbrains.kotlin:kotlin-stdlib:${kotlinVersion}")
```

Reply:
If I change the ```compileOnly``` to ```testImplementation```, Kotlin's reflective functions like ```T::class.java``` wont compile and will give the following error: 
```T::class.java "Unresolved Reference: Java"```. 

But I've changed the ```testCompile``` to ```testImplementation```for the test dependency.

I had used ```compileOnly``` for the Kotlin dependency because the [JUnit Jupiter API](https://github.com/junit-team/junit5/blob/master/junit-jupiter-api/junit-jupiter-api.gradle#L6) is doing it this way.

Reply:
Well, once you switch to the `assertThrows` extension function you don't need `.java` anymore. Don't worry though, I will polish that.